public/blog/github-trending-tab.md

```diff
@@ -0,0 +1,45 @@
+
+For such a popular industry leaderboard, it’s always a challenge to be fair and pick the top N repos most people are happy with. And fair is even more subjective in the developer world.
+* Go vs. Rust?
+* React vs. Vue?
+* MySQL vs. PostgreSQL?
+* Vim vs. Emacs?
+
+Developers are often polarized and strongly opinionated. You are always pissing off at least half the population, even for doing nothing. Open source maintainers often question why other seemingly mediocre repos appear on the ladder while their awesome projects stay in the dark, unnoticed, after spending years of sweat.
+
+And apparently, the GitHub folks don’t have enough capacity to clean up the weed. The GitHub Trending algorithm is indeed flawed, but abruptly shutting down the entire service seems like throwing the bath water out with the baby.
+
+## Star History
+
+star-history.com runs a Twitter account [@StartHistoryHQ](https://twitter.com/starhistoryhq) posting interesting GitHub repos every day.
+
+![star-history](/blog/assets/github-trending-tab/star-history.webp)
+
+Some people DMed us asking if we use a bot to find those repos. Actually, we curate each repo manually, and GitHub Trending Tab is our primary source to find those repos. For us, the Trending Tab serves as a starting point. We review the list every day and pick 1 or 2 interesting repos. Without GitHub Trending, finding the needles in the haystack would require much more effort.
+
+As someone promoting others’ repositories daily, we can sense the strong feelings around GitHub Trending. Because we often see people get excited about us promoting their open source projects, imagine how excited it would be if their projects show up on the GitHub Trending Tab. 
+
+People often complain about the ranking mystery, but mystery also brings serendipity.
+
+## Community
+
+While composing this post, I just checked [github.com](https://github.com), and they still have a dedicated Community tab. To me, Community is what GitHub differentiates itself from other VCS providers like GitLab, Bitbucket.
+
+![github-home](/blog/assets/github-trending-tab/github-home.webp)
+
+GitHub calls itself the home for the Developers. The stars, the issues, the discussions, the followers, the sponsors, and the trendings, it’s these engagements make developers have emotional bindings with GitHub.
+
+I wish GitHub could publish a detailed blog to explain the rationale behind this decision. The "low usage" reason doesn’t resonate with the [developer community](https://github.com/community/community/discussions/31643) and even sounds like an April fool’s joke. Maybe GitHub doesn’t realize it, but removing GitHub Trending could significantly weaken the emotional bindings between the developers and the platform.
+
+There do exist 3rd party websites that provide similar or even more powerful trending features, but none can match the built-in user experience and community vibe GitHub Trending Tab provides.
```

Comment:
```suggestion
There does exist 3rd party websites that provide similar or even more powerful trending features, but none can match the built-in user experience and community vibe GitHub Trending Tab provides.
```

Reply:
This is do

Reply:
I'd say "There are" or "3rd party websites do exist"... 
"There do exist 3rd party websites" sounds weird imo